examples/management_policies.ipynb

```diff
@@ -0,0 +1,551 @@
+    }
+   ],
+   "source": [
+    "from openassetio_mediacreation.specifications.twoDimensional import \\\n",
+    "    PlanarBitmapImageResourceSpecification\n",
+    "from openassetio_mediacreation.traits.color import OCIOColorManagedTrait\n",
+    "\n",
+    "\n",
+    "image_trait_set = PlanarBitmapImageResourceSpecification.kTraitSet | {OCIOColorManagedTrait.kId}\n",
+    "\n",
+    "helpers.display_result(image_trait_set)\n"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-01-30T18:11:38.164101712Z",
+     "start_time": "2024-01-30T18:11:38.106695007Z"
+    }
+   },
+   "id": "f554b02d92aa8091",
+   "execution_count": 32
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "We first want to know whether the manager is interested in entities of this type, and if so, which of the subset of traits in this set have properties that it can `kWrite` to persistent storage."
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "d5cfdd016ffa3e19"
+  },
+  {
+   "cell_type": "code",
+   "outputs": [
+    {
+     "data": {
+      "text/markdown": "> **Result:**\n> `{'openassetio-mediacreation:color.OCIOColorManaged', 'openassetio-mediacreation:twoDimensional.PixelBased', 'openassetio-mediacreation:managementPolicy.Managed', 'openassetio-mediacreation:content.LocatableContent'}`"
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from openassetio.access import PolicyAccess\n",
+    "\n",
+    "\n",
+    "[write_policy] = manager.managementPolicy([image_trait_set], PolicyAccess.kWrite, context)\n",
+    "\n",
+    "helpers.display_result(write_policy.traitSet())"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-01-30T18:11:38.164577993Z",
+     "start_time": "2024-01-30T18:11:38.149463152Z"
+    }
+   },
+   "id": "15e1be1277309b62",
+   "execution_count": 33
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Once again we see the `Managed` trait, indicating that entities of this kind can be published. In addition, we see that `PixelBased`, `LocatableContent` and `OCIOColorManaged` are traits with associated properties that can be persisted by the manager.\n",
+    "\n",
+    "But what if we're not capable of providing data for all those traits? There are actually two questions here\n",
+    "\n",
+    "1. For a given trait, which of its properties _must_ be filled in for that trait to be published, i.e. which properties are non-optional?\n",
+    "2. For a given entity, which traits _must_ have their required properties filled in for the entity to be published?\n",
+    "\n",
+    "The first question varies by trait and the only way to know is to look at the documentation for that trait.\n",
+    "\n",
+    "The second question is once again determined by the manager's policy, where we wish to ask the manager for the `kRequired` subset of an entity's traits"
```

Comment:
[Fix] We can't be happy with this phrasing, it's very complicated to understand. 

My attempt at this section (including the bit below the example)
```
But what if we're not capable of providing data for all those traits? 
That's not necessarily a deal-breaker, as managers will often only require 
a subset of those traits in order to perform a successful publish, 
being able to deduce or omit the remainder. To ask this of the manager, 
we call `managementPolicy` with the `kRequired` access mode, 
which will give us the subset of traits that are fully mandatory for a publish action.

However, recall that traits are used to define the category of the entity, 
in addition to transporting data. Due to this, one _must_ imbue all the traits
of the entity trait set in the published data, but need not necessarily imbue
all the properties of the non-required traits.

<example> 

Note that even within a single given trait, not all properties are necessarily required. 
Consult the documentation for each specific trait to learn which properties may be optional.

Generally, it is a good idea to imbue all the data you have, as the manager is in
the best place to decide what is does or doesn't want to persist.
```

Reply:
I think that's a bit more confusing. The first paragraph doesn't mention properties, so the second paragraph appears to contradict it.

I'll try to come up with something clearer.

Reply:
Rewrote the top bit and took your suggestion for below the example in 2f798cfd, hope that's clearer?

Reply:
Yeah it is clearer to my eye. Just one thing, 

[Consider][Minor]
```
"So we have two questions. We want to know whether the manager is interested in entities of this type at all. If so, we want to know the subset of traits whose properties can be persisted by the manager."

"We can ask this by calling `managementPolicy` with the `kWrite` access mode:"
```

Should this be "We can ask the first question by calling...."? 
I understand that this begins the process of asking both questions, but I think it's clearer if you imply that there's a second step.

Reply:
I guess it should be "we can ask these questions by calling ..." - since `managementPolicy` answers both.

Reply:
"We can answer these questions" in 9ef7ef6e